packages/terra-date-picker/src/react-datepicker/calendar.jsx

```diff
@@ -469,11 +480,11 @@ export default class Calendar extends React.Component {
       return
     }
     return (
-      <div
+      <button
         className={cx('react-datepicker-today-button')}
         onClick={e => this.props.onSelect(getStartOfDate(now(this.props.utcOffset)), e)}>
         {this.props.todayButton}
-      </div>
+      </button>
```

Comment:
I agree that this should be a button instead of a div. When clicking on the today button, only the top border shows the glow and the other borders appears to be cut off by the popup. Is UX ok with this?

![image](https://user-images.githubusercontent.com/22618655/64438169-3c2c1980-d08d-11e9-945a-4e919bdbffdf.png)

Reply:
There is no "Today" button in the UX callouts I've seen for datepicker so I'm not sure what there vision is for it. I've gone ahead and removed the outline style on focus and instead change the background color slightly on focus.